tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -67,7 +115,8 @@ def test_end_to_end_conversion_local_nocolumns(self, dataframe: Any, keep_local:
             'keep_local': keep_local,
         }
 
-        with pytest.raises(ValueError, match=f'.*is not supported by MDSWriter.*'):
+        # Automatic schema inference for ``properties`` is not available
```

Comment:
If it's an `Array(DoubleType())` why can't we infer it to be np.double, which is np.float64?

Reply:
This is for the "dataframe" which has the "properties" column not automatically inferrable.

Reply:
I'm still not understanding why this is the case...